apps/main/src/lend/hooks/useLendPageTitle.ts

```diff
@@ -0,0 +1,15 @@
+import { useEffect } from 'react'
+
+/**
+ * Custom hook to set the document title for markets pages
+ * @param collateralSymbol - The symbol of the collateral token
+ * @param prefix - The prefix part of the title (e.g., "Create - ...")
+ * @param suffix - The suffix part of the title (e.g., ".. - Curve Llamalend")
```

Comment:
It's actually always the same, if the hook is specific for lend we don't need this arg